MyPiggyBank.Integration.Test/StartupTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using MyPiggyBank.Data;
+using MyPiggyBank.Web.Configuration;
+
+namespace MyPiggyBank.Integration.Test
+{
+    public class StartupTest
+    {
+        public IConfiguration Configuration { get; }
+        public StartupTest(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var connectionStringPiggyTest = Configuration.GetConnectionString(nameof(MyPiggyBankContext));
+
+            Action action = !string.IsNullOrEmpty(connectionStringPiggyTest)
```

Comment:
Ta sekcja jest mega nieczytelna. Rozumiem użycie skróconej instrukcji warunkowej, ale zwykłe "ify" dałyby lepszy pogląd na to, co się dzieje. Dodatkowo - czy użycie var będzie lepsze?

Reply:
Niech będzie. Zamiast tego leci zwykły "if else"